spec/spec_helper.rb

```diff
@@ -66,8 +66,10 @@ def self.ruby=(ruby)
 
   git_version = Bundler::Source::Git::GitProxy.new(nil, nil, nil).version
 
+  config.filter_run_excluding :ruby => RequirementChecker.against(RUBY_VERSION)
   config.filter_run_excluding :rubygems => RequirementChecker.against(Gem::VERSION)
   config.filter_run_excluding :git => RequirementChecker.against(git_version)
+  config.filter_run_excluding :rubygems_master => (ENV["RGV"] != "master")
```

Comment:
The added filters are also probably rebase leftovers? Or is there some compatibility issue with this PR?

Reply:
I missed those, they are leftovers from the rebase I just did probably.